src/core/openthread-core-windows-config.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This is a project config file in that it is specific to a particular platform.  As such I agree with Marcin, that it should reside outside of the OT core.